index.jade

```diff
@@ -19,8 +19,12 @@ html(lang='en')
 
             // reload same-named components when css changes
             hr.on('change', (fileName) => {
+              if (console['clear']) console['clear']();
+              console.log(`---- hot-reloaded ${fileName} ----`);
               if (fileName.match(/app\.component/)) {
                 hr.hotReload('app/app.component.ts');
+              } else if (fileName.match(/storyedit\.forms/)) {
+                hr.hotReload('app/storyedit/directives/edit.component.ts');
```

Comment:
I'm trying to "guess" which ts file to load, based on the name of the css file that changed.  In many cases, they have the same name except for the `css/ts` extension.  But this form css is shared between components, so for faster debugging I manually triggered the refresh.

In other words... that's mostly a debugging-helper.